src/app/savings/savings-account-view/transactions/view-transaction/datatable-transaction-tab/datatable-transaction-tab.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'mifosx-datatable-transaction-tab',
+  templateUrl: './datatable-transaction-tab.component.html',
+  styleUrls: ['./datatable-transaction-tab.component.scss']
+})
+
+export class DatatableTransactionTabComponent implements OnInit {
+  entityId: string;
+  /** Savings Datatable */
+  entityDatatable: any;
+  /** Multi Row Datatable Flag */
+  multiRowDatatableFlag: boolean;
+  constructor(private route: ActivatedRoute) {
+    this.entityId = this.route.parent.parent.snapshot.paramMap.get('id');
+    this.route.data.subscribe((data: { transactionDatatable:any }) => {
+      this.entityDatatable = data.transactionDatatable;
+      this.multiRowDatatableFlag = this.entityDatatable.columnHeaders[0].columnName === 'id' ? true : false;
+    });
+   }
+
+  ngOnInit(): void {
```

Comment:
Is it required if it is empty?

Reply:
No, it is not required I can remove it, actually when we generate a component in angular this comes automatically.